### DISCUSSION TITLE
is there any function like ethersjs getBlockWithTransactions

### DISCUSSION THREAD

**Discussion by EasyChris** [wevm/viem#1362] [Question] [answered]
is there any function like ethersjs getBlockWithTransactions  

https://docs.ethers.org/v5/api/providers/provider/#Provider-getBlockWithTransactions

that i can get custom  block number and then get  all this block  transactions

**Comment by tmm** [comment, marked as answer]
https://viem.sh/docs/actions/public/getBlock.html#includetransactions-optional
